### PR TITLE
[JBPM-8454] Stabilize integration test StandaloneControllerMultinodeIT

### PR DESCRIPTION
**Pull request by elguardian**
deploying manually as managed deployment causes race conditions.

### CODE REVIEW

**Comment by elguardian** [comment]
ok to test.

**Comment by elguardian** [comment]
@jomarko as you reviewed #2285 do you mind to check out on this ?

**Comment by jomarko** [comment]
@elguardian sure, however the touched test fails on jenkins.

**Comment by elguardian** [comment]
this is ok to test.

**Comment by elguardian** [comment]
retest this please

**Comment by elguardian** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
@jomarko finally make it through. this morning were some problems with the build

File: kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/test/java/org/kie/workbench/common/screens/server/management/backend/StandaloneControllerMultinodeIT.java [hunk 1]
```diff
@@ -157,14 +162,34 @@ public void onContainerSpecUpdated(ContainerSpecUpdated containerSpecUpdated) {
...
+                templates = client.listServerTemplates();
+                LOGGER.info("try #{} templates", ++count);
+            }
+
             ServerInstanceKeyList list = client.getServerInstances(KIE_SERVER_ID);
+            while (list.getServerInstanceKeys().length == 0) {
+                Thread.sleep(500L);
+                list = client.getServerInstances(KIE_SERVER_ID);
+                LOGGER.info("try #{} instances", ++count);
+            }
```
Comment (jomarko) [inline kie-wb-common-screens/kie-wb-common-server-ui/kie-wb-common-server-ui-backend/src/test/java/org/kie/workbench/common/screens/server/management/backend/StandaloneControllerMultinodeIT.java]: @elguardian thank you for updates. However I miss some timeout in these two `while` cycles. I think we do not need to add them and we can reuse the existing event handler. See my proposal of the solution https://github.com/elguardian/kie-wb-common/pull/1.

**Comment by elguardian** [comment]
Jenkins retest this please

**Comment by elguardian** [comment]
> @elguardian thank you. Approving. However I think need to be rerun by jenkins before merge :(

I did in local

**Comment by jomarko** [comment]
@kiegroup/gatekeepers it is in your hands.

**Comment by manstis** [comment]
I ran locally however received this error:
```
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   StandaloneControllerMultinodeIT.org.kie.workbench.common.screens.server.management.backend.StandaloneControllerMultinodeIT » Runtime
[INFO] 
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```
The fail-safe report is:
```
-------------------------------------------------------------------------------
Test set: org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 8.74 s <<< FAILURE! - in org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT
org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT  Time elapsed: 8.74 s  <<< ERROR!
java.lang.RuntimeException: Could not invoke deployment method: public static org.jboss.shrinkwrap.api.spec.WebArchive org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT.createEmbeddedControllerWarDeployment()
	at org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT.createEmbeddedControllerWarDeployment(EmbeddedControllerIT.java:69)
Caused by: org.eclipse.aether.resolution.DependencyResolutionException: Cannot access jboss-public-repository-group (https://repository.jboss.org/nexus/content/groups/public/) in offline mode and the artifact commons-jxpath:commons-jxpath:jar:1.3 has not been downloaded from it before.
	at org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT.createEmbeddedControllerWarDeployment(EmbeddedControllerIT.java:69)
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Cannot access jboss-public-repository-group (https://repository.jboss.org/nexus/content/groups/public/) in offline mode and the artifact commons-jxpath:commons-jxpath:jar:1.3 has not been downloaded from it before.
	at org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT.createEmbeddedControllerWarDeployment(EmbeddedControllerIT.java:69)
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Cannot access jboss-public-repository-group (https://repository.jboss.org/nexus/content/groups/public/) in offline mode and the artifact commons-jxpath:commons-jxpath:jar:1.3 has not been downloaded from it before.
	at org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT.createEmbeddedControllerWarDeployment(EmbeddedControllerIT.java:69)
Caused by: org.eclipse.aether.transfer.RepositoryOfflineException: Cannot access jboss-public-repository-group (https://repository.jboss.org/nexus/content/groups/public/) in offline mode
	at org.kie.workbench.common.screens.server.management.backend.EmbeddedControllerIT.createEmbeddedControllerWarDeployment(EmbeddedControllerIT.java:69)
```
Are there any prerequisite I need to run successfully?

**Comment by manstis** [comment]
There were other failing tests (I assume for the same reason).

**Comment by mareknovotny** [comment]
Jenkins retest this

**Pull request merged**